kong/db/schema/init.lua

```diff
@@ -6,6 +6,7 @@ local is_reference = require "kong.pdk.vault".is_reference
 local json         = require "kong.db.schema.json"
 local cjson_safe   = require "cjson.safe"
 local deprecation  = require "kong.deprecation"
+local inspect      = require "inspect"
```

Comment:
I'm also a little torn on the usage of "inspect" in here. Is there precedence behavior that justifies it?

Reply:
I don't think there is. Honestly I don't like it that much either. Maybe let's just remove it for the sake of simplicity. 🤔

Reply:
Removed ✅